pkg/addon/manager/addon.go

```diff
@@ -207,7 +228,7 @@ func newAddonAgentRole(namespace string) *rbacv1.Role {
 			{
 				APIGroups: []string{""},
 				Resources: []string{"serviceaccounts", "serviceaccounts/token"},
-				Verbs:     []string{"create"},
+				Verbs:     []string{"get", "watch", "list", "create", "delete"},
```

Comment:
nit: how about "*"?

Reply:
now the local SAs are cached by list-watching, so we dont need get. lets polish it in a follow-up